### PR TITLE
Issue #11 Added option for not connecting to the internet

### PR DESCRIPTION
**Pull request by DarkSouL11**

### CODE REVIEW

File: UIP.py [hunk 1]
```diff
@@ -1,4 +1,6 @@
 from scheduler import scheduler
 
 if __name__ == "__main__":
-	scheduler()
\ No newline at end of file
+  print("Press 1 to connect to internet or any other key to use this application in offline mode")
+  conn = int(input()) == 1
```
Comment (abhsag24) [inline UIP.py]: can we rename this variable to offline? i have some huge plans for this :P

File: scheduler.py [hunk 2]
```diff
@@ -6,20 +6,24 @@
...
+            while not os.path.isdir(os.path.join(CURR_DIR,PICS_FOLDER)):
+                print('Downloading images..')
+                time.sleep(60)
+        elif not os.path.exists(directory):
+                os.makedirs(directory)
         self.change_random()
         self.setStartTime(time.time())
         self.changeCycle()
-        
+
```
Comment (abhsag24) [inline scheduler.py]: generally such unrelated changes are made into new commits but i guess this fine for now :)

**Comment by abhsag24** [comment]
@DarkSouL11 we don't create new commits to change things of the previous commit. Can you squash these two commits together?

**Comment by DarkSouL11** [comment]
How to squash two commits together?

**Comment by abhsag24** [comment]
whew awesome work buddy!

**Pull request merged**